HHS/Head-Start-TTADP
# [PROD] Reset table pagination when filters change

## Description of change
If a filter is changed on the activity report table component, then the active page should be reset to zero. This is true of any of the three pages where this component lives that also have filters.

## How to test
Visit [the staging environment](https://tta-smarthub-staging.app.cloud.gov). Select a filter on a page with an activity reports table, change the page. Update the filters. Notice that the page is reset to the first.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1444

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
